src/WorkflowManager/Program.cs

```diff
@@ -112,11 +112,6 @@ private static void Main(string[] args)
         {
             var host = CreateHostBuilder(args).Build();
 
-            if (args.Length == 0)
```

Comment:
This change will mean the TaskManager wont start when the app is run ?

Reply:
Remember when you spotted something like the task manager running twice? This was the cause of it. It was starting it for a second time. It is already started in the AddTaskManager step.